Src/WitsmlExplorer.Frontend/components/ContextMenus/TrajectoryContextMenu.tsx

```diff
@@ -32,23 +32,28 @@ const TrajectoryContextMenu = (props: ObjectContextMenuProps): React.ReactElemen
     dispatchOperation({ type: OperationType.HideContextMenu });
   };
 
+  const trajectoryMenuItems = (): React.ReactElement[] => {
```

Comment:
To ensure consistency, I suggest naming this extraMenuItems as well.

Reply:
fixed